shell/browser/api/atom_api_web_contents.cc

```diff
@@ -124,15 +124,34 @@
 
 namespace gin {
 
+std::string GetPrinterDescription(int status) {
+  switch (status) {
+    case 0:
+      return "idle";
+    case 1:
```

Comment:
is this intentionally falling-through? If so, please use `FALLTHROUGH`.
```suggestion
    case 1:
      FALLTHROUGH;
```